sdk/dotnet/Pulumi.Automation/WorkspaceStack.cs

```diff
@@ -530,7 +530,7 @@ void OnPreviewEvent(EngineEvent @event)
         /// <summary>
         /// Gets the current set of Stack outputs from the last <see cref="UpAsync(UpOptions?, CancellationToken)"/>.
         /// </summary>
-        private async Task<ImmutableDictionary<string, OutputValue>> GetOutputAsync(CancellationToken cancellationToken)
+        public async Task<ImmutableDictionary<string, OutputValue>> GetOutputAsync(CancellationToken cancellationToken = default)
```

Comment:
Do we want to keep the singular name `GetOutputAsync` or use plural `GetOutputsAsync`? Other APIs have `outputs`

Reply:
Let's change it to `GetOutputs` to align with the other SDKs and more accurately reflect the functionality.